common/rfb/H264LibavDecoderContext.cxx

```diff
@@ -114,6 +128,15 @@ rdr::U8* H264LibavDecoderContext::validateH264BufferLength(rdr::U8* buffer, rdr:
 }
 
 void H264LibavDecoderContext::decode(rdr::U8* h264_buffer, rdr::U32 len, rdr::U32 flags, ModifiablePixelBuffer* pb) {
+  os::AutoMutex lock(&mutex);
+  if (stale)
+  {
+    freeCodec();
+    return;
+  }
+  if (!initialized)
+    return;
+  decoding = true;
```

Comment:
This variable seems to be unused?

Reply:
You are right. It was a part of some earlier state-machine but right now it is unused. removed.